lib/rouge/lexers/sml.rb

```diff
@@ -41,9 +41,7 @@ def self.analyze_text(text)
         rule /[(\[{]/, Punctuation, :main
         rule /[)\]}]/, Punctuation, :pop!
 
-        rule /\b(let|if|local)\b(?!')/, Keyword::Reserved do
-          push; push
```

Comment:
FYI: It's possible I'm misunderstand the point of this `push; push`, but it was omitting the `left`, `if`, and `local` keywords. Perhaps it was just a typo?